ambientweather2sqlite/daemon.py

```
@@ -25,21 +25,22 @@
+    database: DatabaseManager,
+    labels: dict[str, str],
     *,
     port: int | None = None,
     period_seconds: int = 60,
 ) -> None:
     print(f"Observing {live_data_url}")
     print("Press Ctrl+C to stop")
-    labels, _ = create_metadata(database_path, live_data_url)
+    
 
     server = None
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)